tests/Fuerte/ConnectionConcurrentTest.cpp

```diff
@@ -146,12 +161,14 @@ TEST_P(ConcurrentConnectionF, CreateDocumentsParallel) {
       }
     });
   }
-  ASSERT_TRUE(wg.wait_for(
+  ASSERT_TRUE(wg->wait_for(
       std::chrono::seconds(300)));  // wait for all threads to return
 
   // wait for all threads to end
   joins.joinAll();
 
+  guard.fire();
```

Comment:
NIT
Probably not necessary?
Because if we didn't wait_for it. We don't execute code after assert_true
But if assert_true, is it unnecessary?